be/src/exprs/vectorized/runtime_filter_bank.h

```diff
@@ -192,6 +233,18 @@ class RuntimeFilterProbeCollector {
     void set_plan_node_id(int id) { _plan_node_id = id; }
     int plan_node_id() { return _plan_node_id; }
 
+    // sort descriptors to make descriptors which have the same partition_by_exprs adjacent.
+    void do_sort_descriptors_if_needed(bool is_sort) {
```

Comment:
two duplicate do_sort_descriptor_if_needed function, just keep only one,
suggest to use sort_descriptor_if_needed instead of do_sort_descriptor_if_needed (remove do_)

Reply:
I will later support multi columns runtime into one, so I delete this codes in this pr.